android/expoview/build.gradle

```diff
@@ -74,7 +74,7 @@ android {
     versionCode 1
     versionName "1.0"
     ndk {
-      abiFilters 'armeabi-v7a', 'x86'
+      abiFilters 'armeabi-v7a', 'x86', 'arm64-v8a', 'x86_64'
```

Comment:
Shouldn't we also change something in `ReactAndroid`?

Reply:
react native is supporting 64 bits so  we don't need to change anything there